### ISSUE TITLE
No visual label provided for query field and programmatic name is misaligned with field purpose: A11y_DocumentDB for VS Code Extension_View Query Insights_Lable in Name

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-documentdb#379]
### GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #FTP; #Win32; #DocumentDB for VS Code Extension; #SH_DocumentDB_Win32_NOV2025; #WCAG2.5.3; #Lable in Name;

### Environment Details:
Application Name: DocumentDB for VS Code Extension
Version 0.6.0
OS: Windows 11[Version 23H2(OS Build 22631.3593)]

### Repro Steps:

1. Open VS code
2. Install the DocumentDB for VS Code Extension and open it
3. Navigate to the DocumentDB Connections area.
4. Click the New Connection action.
5. Select Connection String as the connection method.
6. Paste the connection string when prompted.
7. Choose to authenticate with username/password and enter the credentials
8. Confirm to create the connection.
9. Locate the desired cluster in the DocumentDB Connections list.
10. Expand the cluster entry to initiate the connection.
11. Expand one of the databases under the cluster.
12. Expand a collection within that database.
13. Select the Documents item for that collection.
14. Collection View will open
15. Observe the issues No visual label provided for query field and programmatic name is misaligned with field purpose

### Actual Result:
No visual label provided for query field and programmatic name is misaligned with field purpose

### Expected Result:
Provide a Visual Label: Add a visible label to the query field and programmatically associate it to the field.
Ensure the accessible name matches the visual label. Avoid using an aria-label that conflicts with the field's purpose. Example:

### User Impact:
Users relying on screen readers cannot identify the purpose of the query field due to the lack of a visual label and an incorrect programmatic name. This creates confusion and prevents users from interacting with the field effectively.
Users with cognitive disabilities may struggle to understand the field's purpose without a clear visual label or meaningful instructions.

### Attachment

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/b435d937-aecc-4534-84aa-9084a54eef95" />

**Comment by tnaum-ms** [comment]
Reopening after a github automation. Ready for review once `0.7.0` is released.

**Comment by AnKushSingh05** [comment]
#Closed;#A11yFixed;
Issue is fixed

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/5b7195aa-1ee8-41d2-8f75-f11f4b63b2d9" />
